core/validation/messages.py

```diff
@@ -8,6 +8,7 @@
 )
 BLANK_ZERO = BLANK + " Enter a value, even if it’s zero."
 BLANK_PSI = BLANK + " Enter why the private sector investment gap is greater than zero."
+BLANK_SIGN_OFF = "In tab '8 - Review & Sign-Off', cell {cell} is blank but is required."
```

Comment:
Could this be moved to the bottom of the file under a comment highlighting it as a pre-transformation message?